bot/seasons/evergreen/speedrun.py

```diff
@@ -0,0 +1,29 @@
+import json
+import logging
+from pathlib import Path
+from random import choice
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Speedrun(commands.Cog):
+    """Commands about the video game speedrunning community."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="speedrun")
+    async def get_speedrun(self, ctx):
+        """Sends a link to a video of a random speedrun."""
+        with open(Path('bot/resources/evergreen/speedrun_links.json')) as file:
```

Comment:
To be safe, the encoding should be explicitly specified. Also, I think `Path().open()` is cleaner than `open(Path())`

```suggestion
        with Path('bot/resources/evergreen/speedrun_links.json').open(encoding="utf-8") as file:
```